paket.dependencies

```diff
@@ -5,10 +5,12 @@ group Main
   source https://api.nuget.org/v3/index.json
   storage:none
 
-  nuget FSharp.Core >= 4.7.2
+  nuget FSharp.Core 5.0.0
   nuget Ply 0.3.1
   nuget Npgsql 5.0.3
 
+  nuget DotNet.ReproducibleBuilds copy_local: true
```

Comment:
No worries, I didn't know about this package, that will simplify my projects :)

Reply:
Actually, it seems to break when I use it on Azure pipelines (with a GitHub source repo).
It seems that having either of the two lines:
```xml
<DeterministicSourcePaths>true</DeterministicSourcePaths>
<ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
```
breaks SourceLink for F# projects.

Reply:
do you have any logs or anything that would help triaging this?  I ask because `dotnet validate`, which validates sourcelink-ability of the packages, returns valid for this package after this change:

```
➜  Debug git:(add-interpolated-strings) ✗ dotnet validate package local Npgsql.FSharp.4.0.0.nupkg
Validating /Users/chethusk/oss/Npgsql.FSharp/src/bin/Debug/Npgsql.FSharp.4.0.0.nupkg
• Source Link: ✅ Valid
```

Reply:
The problem only comes up during Continuous Integration build.
To reproduce it locally, you can set the environment variable `TF_BUILD` to `true` (which in effect enables the two lines above).
But agreed, on local build it works (that fooled me too)